cli/mobycli/exec.go

```diff
@@ -112,7 +112,10 @@ func IsDefaultContextCommand(dockerCommand string) bool {
 }
 
 // ExecSilent executes a command and do redirect output to stdOut, return output
-func ExecSilent(ctx context.Context) ([]byte, error) {
-	cmd := exec.CommandContext(ctx, ComDockerCli, os.Args[1:]...)
+func ExecSilent(ctx context.Context, args ...string) ([]byte, error) {
+	if len(args) == 0 {
```

Comment:
How was this change required ? maybe we should change the caller to make sure args include the `docker` command

Reply:
This is used on `getOutFromMoby` when shelling out to moby. The idea is to hack the `format` option before passing it to moby, since it should treat the cases of `pretty` and `json`. Both are inexistent on moby, so I do a little trick to translate it.